### PR TITLE
Fix gps rescue throttle pid gains

### PR DESCRIPTION
**Pull request by ctzsnooze**
This PR is a simple one to:
- update the GPS Rescue Altitude control Debug traces (PID and altitude),
- add GPS Rescue Velocity control Debug traces (PID and velocity), and
- update the RTH graphs and labels.

It is intended to properly display the new GPS Rescue debug traces from [11579](https://github.com/betaflight/betaflight/pull/11579)

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — DISMISSED** [review]

File: js/graph_config.js
Comment (haslinghuis) [inline js/graph_config.js:755]: Same

File: js/graph_config.js
Comment (haslinghuis) [inline js/graph_config.js:781]: Best to move this line to 775 (and remove the inner block) as fall through case as the inner block is repeated.

File: js/graph_config.js
Comment (haslinghuis) [inline js/graph_config.js:788]: Same.

File: js/graph_config.js
Comment (haslinghuis) [inline js/graph_config.js:795]: Same.

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Please remove duplicate blocks (to remove the code duplication errors).

File: js/flightlog_fields_presenter.js
Comment (McGiverGim) [inline js/flightlog_fields_presenter.js:1092]: I suppose that `debug 3` is wrong and is `debug 2`. Move the comments to the default to let clear where it will enter. For example:
```suggestion
                    switch (fieldName) {
                        case 'debug[3]': // current altitude in m
                            return (value / 100).toFixed(2) + 'm';                            

                        // debug 0 = Throttle P
                        // debug 1 = Throttle I
                        // debug 2 = Throttle D                            
                        default:
                            return value.toFixed(0);
                    }
```

File: js/flightlog_fields_presenter.js
Comment (McGiverGim) [inline js/flightlog_fields_presenter.js:1170]: Same than before:
```suggestion
                    switch (fieldName) {
                        case 'debug[3]': // Velocity cm/s
                            return (value / 100).toFixed(2) + 'm/s';

                        // debug 0 = Velocity P
                        // debug 1 = Velocity I
                        // debug 2 = Velocity D
                        default:
                            return value.toFixed(0);
                    }
```

File: js/flightlog_fields_presenter.js
Comment (McGiverGim) [inline js/flightlog_fields_presenter.js:1186]: The same, group cases, and some comments seems wrong from a copy&paste.
```suggestion
                case 'GPS_RESCUE_HEADING':
                    switch (fieldName) {
                        case 'debug[0]': // error angle tenths of a degree
                        case 'debug[1]': // angle to home in tenths of a degree                        
                            return (value / 10).toFixed(1) + 'deg';
                        case 'debug[2]': // commanded yaw rate in tenths of a degree
                            return (value / 10).toFixed(1) + 'deg/s';
                        case 'debug[3]': // roll angle in hundredths of a degree
                            return (value / 100).toFixed(2) + 'deg';
                        default:
                            return value.toFixed(0);
                    }
```

**Review by McGiverGim — CHANGES_REQUESTED** [review]
My comments go in the same line than @haslinghuis. Try to group `case` elements, in this way is clear that various elements have exactly the same behaviour. For the rest the PR is ok.

**Comment by ctzsnooze** [comment]
Not sure if I addressed all the issues above, if not please let me know.

Added another debug for https://github.com/betaflight/betaflight/pull/11579, tested.

**Comment by McGiverGim** [comment]
> Not sure if I addressed all the issues above, if not please let me know.

I think my review has not been fixed, I don't see any change in the code about it.

File: js/flightlog_fields_presenter.js
Comment (McGiverGim) [inline js/flightlog_fields_presenter.js:1094]: the `uS` unit here is different from line 1031 or 1035, where is lowercase. I think all of them must be equal.
```suggestion
                    switch (fieldName) {
                        case 'debug[0]': // Throttle P added us
                        case 'debug[1]': // Throttle D added * us
                            return value.toFixed(0) + 'us';
```

File: js/flightlog_fields_presenter.js
Comment (McGiverGim) [inline js/flightlog_fields_presenter.js:1038]: Is better to have them in different order to not duplicate code:
```suggestion
                    switch(fieldName) {
                        case 'debug[0]':
                        case 'debug[2]':                        
                            return value.toFixed(0) + 'us';
                        case 'debug[1]':
                            return (value / 100).toFixed(1) + 'deg';
                        default:
                            return value.toFixed(0);
                    }
```

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: js/flightlog_fielddefs.js
Comment (McGiverGim) [inline js/flightlog_fielddefs.js:345]: Sonar issue: missing trailing comma

File: js/flightlog_fields_presenter.js
Comment (McGiverGim) [inline js/flightlog_fields_presenter.js:1180]: Sonar issue: remove the trailing space at the end

**Review by McGiverGim — CHANGES_REQUESTED** [review]
Apart from my other review, Sonar is complaining about some things, I've marked the basic. The others, I think we can ignore them.

File: js/flightlog_fielddefs.js
Comment (ctzsnooze) [reply js/flightlog_fielddefs.js:345]: thanks, fixed

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js:1094]: thanks, done

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js:1180]: thanks

File: js/graph_config.js
Comment (ctzsnooze) [reply js/graph_config.js]: I tried to fix this, hopefully it is OK now

File: js/graph_config.js
Comment (ctzsnooze) [reply js/graph_config.js]: Hopefully OK now

File: js/graph_config.js
Comment (ctzsnooze) [reply js/graph_config.js]: OK now I hope

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js]: Thanks, have done as suggested.

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js]: OK thanks

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js]: thanks, updated

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js]: Have revised these and hopefully is OK now.

File: js/flightlog_fields_presenter.js
Comment (McGiverGim) [inline js/flightlog_fields_presenter.js:1037]: I think this is the only one missing. The debug[0] and debug[2] are the same, so group this two and I think we are ready.

**Review by McGiverGim — CHANGES_REQUESTED** [review]
The latest one, please ;)

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js]: Thank you.  I hope it is fixed now.
Should I squash?

File: js/graph_config.js
Comment (McGiverGim) [inline js/graph_config.js:783]: Sonar issue: trailing space at the end

**Review by McGiverGim — COMMENTED** [review]
I think is ok now. A missing trailing space, but for the rest is ok to me. If you squash the commits, please remove the trailing space too ;)

**Review by haslinghuis — APPROVED** [review]

**Comment by ctzsnooze** [comment]
Closing, replaced with #593

**Pull request closed without merging**
